apps/prairielearn/src/pages/instructorAssessmentAccess/accessRulesTable.ts

```diff
@@ -0,0 +1,245 @@
+      data-timezone="${timezone}"
+    >
+      <table class="table table-sm">
+        <thead>
+          <tr id="tableHeaderRow">
+            <th id="arrowButtonsHeader" ${editMode ? '' : 'hidden'}></th>
+            <th id="editButtonHeader" ${editMode ? '' : 'hidden'}></th>
+            <th id="deleteButtonHeader" ${editMode ? '' : 'hidden'}></th>
+            <th>Mode</th>
+            <th>UIDs</th>
+            <th>Start date</th>
+            <th>End date</th>
+            <th>Active</th>
+            <th>Credit</th>
+            <th>Time limit</th>
+            <th>Password</th>
+            <th>PrairieTest</th>
+          </tr>
+        </thead>
+        <tbody>
+          ${accessRules.map((access_rule, index) => {
+            // Only users with permission to view student data are allowed
+            // to see the list of uids associated with an access rule. Note,
+            // however, that any user with permission to view course code
+            // (or with access to the course git repository) will be able to
+            // see the list of uids, because these access rules are defined
+            // in course code. This should be changed in future, to protect
+            // student data. See https://github.com/PrairieLearn/PrairieLearn/issues/3342
+            return html`
+              <tr>
+                <td class="arrowButtonsCell align-content-center" ${editMode ? '' : 'hidden'}>
+                  <div>
+                    <button
+                      class="btn btn-xs btn-secondary up-arrow-button"
+                      type="button"
+                      data-row="${index}"
+                      ${index === 0 ? 'disabled' : ''}
+                    >
+                      <i class="fa fa-arrow-up" aria-hidden="true"></i>
+                    </button>
+                  </div>
+                  <div>
+                    <button
+                      class="btn btn-xs btn-secondary down-arrow-button"
+                      type="button"
+                      data-row="${index}"
+                      ${index === accessRules.length - 1 ? 'disabled' : ''}
+                    >
+                      <i class="fa fa-arrow-down" aria-hidden="true"></i>
+                    </button>
+                  </div>
+                </td>
+                <td class="editButtonCell align-content-center" ${editMode ? '' : 'hidden'}>
+                  <button
+                    class="btn btn-sm btn-secondary editButton"
+                    type="button"
+                    data-row="${index}"
+                    data-toggle="modal"
+                    data-target="editAccessRuleModal"
+                    data-access-rule-mode="${access_rule.assessment_access_rule.mode}"
+                    data-access-rule-uids="${access_rule.assessment_access_rule.uids
+                      ? access_rule.assessment_access_rule.uids.join(', ')
+                      : ''}"
```

Comment:
Nit, more concisely:

```suggestion
                    data-access-rule-uids="${access_rule.assessment_access_rule.uids?.join(', ') ?? ''}"
```

Reply:
This has been removed entirely in [660f81b](https://github.com/PrairieLearn/PrairieLearn/pull/9934/commits/660f81bebe37d51fcef94b1535961f4abfa02d88).